tidy3d/components/medium.py

```diff
@@ -322,53 +324,67 @@ def _validate_numiters(cls, val):
             )
         return val
 
-    @property
-    def complex_fields(self) -> bool:
-        """Whether the model uses complex fields."""
-        return False
-
 
 class TwoPhotonAbsorption(NonlinearModel):
     """Model for two-photon absorption (TPA) nonlinearity which gives an intensity-dependent
     absorption of the form :math:`\\alpha = \\alpha_0 + \\beta I`.
     Also includes free-carrier absorption (FCA) and free-carrier plasma dispersion (FCPD) effects.
     The expression for the nonlinear polarization is given below.
 
-    Note
+    Notes
     ----
```

Comment:
In RST you need the dashes underneath to be at least as long as the title, so add one more dash.